executors/amqp/README.md

```diff
@@ -0,0 +1,72 @@
+# Venom - Executor AMQP
+
+Step to publish / subscribe to AMQP 1.0 compatible broker (currently implemented by QPID and ActiveMQ, among others)
+
+## Input
+
+```yaml
+- Addr         (address of the amqp broker to connect to, in the format "amqp://<host>:<port>")
+- ClientType   (consumer or producer)
+
+# Consumer Parameters
+- SourceAddr   (source topic/queue to consume messages from)
+- MessageLimit (number of messages to read from the broker before returning result)
+
+# Producer Parameters
+- TargetAddr   (topic/queue to which messages should be published)
+- Messages     (array of message bodies to send to the broker)
```

Comment:
```suggestion
- addr         (address of the amqp broker to connect to, in the format "amqp://<host>:<port>")
- clientType   (consumer or producer)

# Consumer Parameters
- sourceAddr   (source topic/queue to consume messages from)
- messageLimit (number of messages to read from the broker before returning result)

# Producer Parameters
- targetAddr   (topic/queue to which messages should be published)
- messages     (array of message bodies to send to the broker)
```

Reply:
Thanks @fsamin - not sure how I missed this!